istio-control/istio-discovery/templates/configmap.yaml

```diff
@@ -211,5 +211,13 @@ data:
       envoyAccessLogService:
         address: {{ .Values.global.proxy.envoyAccessLogService.host }}:{{ .Values.global.proxy.envoyAccessLogService.port }}
     {{- end}}
+
+    {{- if .Values.pilot.enableCertificateController }}
+      certificates:
+        - secretName: istio-webhook-galley
```

Comment:
Let's not use random names for the secretName. 
Istio spiffe secrets follow the pattern: istio.SERVICE_ACCOUNT_NAME

For the DNS certs we should follow a similar convention - istiodns or dns prefix ( I would use dns. to make it clear it's not istio specific - unlike the Spifee accounts that are specific to Istio due the SAN format ). And the second part should be based on either the service account or service name. 

While this won't be enforced, it's good to be organized and have a good style.

Reply:
Done: addressed in https://github.com/istio/installer/pull/424; please review it.